create the security page.

nisarhassan12 opened the pull request:
Fixes #938 
This is how it looks:

![image](https://user-images.githubusercontent.com/46004116/133144657-e7c4031c-3c74-4e83-8248-4129a2a83191.png)


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/967"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/967"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

mikenikles commented:
I see #937 as an another PR that claims to add a security page. Is #937 obsolete if we merge this one here?

ChristinFrohne commented on src/contents/security.ts:
```diff
@@ -0,0 +1,36 @@
+	{
+		title: 'Compliance',
+		paragraphs: ['Gitpod is a European company committed to developer and data privacy. We are GDPR compliant and provide our users with the ability to access and control the information that we collect about them.', 'Gitpod is built with security in mind and we continuously invest in security best practices. We are currently in the process of becoming SOC 2 compliant and you can request a copy of  ...
+		image: 'compliance.svg',
```
is this the latest version of the GDPR icon? The alignment of the stars seems off

nisarhassan12 replied:
Nope. It was not I have updated it now.

ChristinFrohne reviewed (commented):
Great, thanks! How did you decide on the choice of logos? They are partly different from the mockup. Once we updated the landing page these logos should be in sync with that

Outstanding to-do's:
- [x] add a link to the footer

- [ ] adjust the mobile view
<img width="368" alt="Bildschirmfoto 2021-09-15 um 09 44 38" src="https://user-images.githubusercontent.com/46345125/133392104-25006325-0bde-4081-ac95-809615f1d1ca.png">

ChristinFrohne commented:
I thought #937 would be merged first and this Pr would update the layout. 
If we discard #937, this page (https://deploy-preview-937--gitpod-kumquat.netlify.app/security/report) needs to be included in this PR as well. Instead of the big call-out section at the end of /report, we should use the "thanks" section that is used before.
Could you add that @nisarhassan12 ?

nisarhassan12 commented:
> Could you add that @nisarhassan12 ?
@ChristinFrohne Done ✅  

I have also addressed the other feedback as well. Please have a look again. 

One thing to watch out for is the page titles that I added here https://github.com/gitpod-io/website/pull/967/commits/93a31cf887a348752d6bd73b38d43fea0fe139aa Thanks

ChristinFrohne commented:
Thanks!
- sorry I should have been more clear about the "thanks" sections on /report. `security/report` should link to /`security/thanks` by reusing this section:
<img width="311" alt="Bildschirmfoto 2021-09-15 um 13 49 51" src="https://user-images.githubusercontent.com/46345125/133428089-ab970555-b497-40fa-93da-5d672fef4716.png">

- Did you see my former comment? "How did you decide on the choice of logos? They are partly different from the mockup. Once we updated the landing page these logos should be in sync with that"
- could you update the tab headline to "Security Disclosure Policy"
<img width="249" alt="Bildschirmfoto 2021-09-15 um 13 55 22" src="https://user-images.githubusercontent.com/46345125/133429197-dcd4e7cd-70b6-4d87-a6d5-987b7f3a4d94.png">

Once you have addressed the feedback, please ping Geoff to give the messaging the final tweak.

nisarhassan12 commented:
> Did you see my former comment? "How did you decide on the choice of logos? They are partly different from the mockup. Once we updated the landing page these logos should be in sync with that"

Sorry I read that earlier but somehow it slipped from my mind, Now I have updated the order this is how it looks:

![image](https://user-images.githubusercontent.com/46004116/133433157-7bda4ac0-5709-4b60-9279-176c3ef381a3.png)

Please note that the first row now has the same order both on the Home and Security page.

nisarhassan12 commented:
> sorry I should have been more clear about the "thanks" sections on /report. security/report should link to /security/thanks by reusing this section:

Thanks. Done this is how it looked

![image](https://user-images.githubusercontent.com/46004116/133558126-197a51f7-afaa-4f07-abbc-10134fa3c924.png)

which looks different than the rest of the white boxes we have in isolation on different pages so I have updated it to look like the following:

![image](https://user-images.githubusercontent.com/46004116/133558461-6ace2f99-38b4-475e-b60c-a1e80022d11a.png)

@ChristinFrohne Are you fine with this change ? Note that this change only applies to the report page and not the security page. Thanks

ChristinFrohne commented:
Great initiative @nisarhassan12 ! Looks great ✨

ghuntley reviewed (approved):
Can't wait to have this shipped. Ready to go from my perspective.

JohannesLandgraf commented:
It is really important that we take ownership seriously. We cannot merge company policies without the consent of the DRI for that topic. In this case this is @meysholdt who needs to approve this PR before it goes live.
